packages/api/cms-api/src/dam/files/files.resolver.ts

```diff
@@ -141,7 +143,10 @@ export function createFilesResolver({ File, Scope: PassedScope }: { File: Type<F
             return this.filesService.moveBatch(files, targetFolder);
         }
 
-        @Mutation(() => CopyFilesResponse)
+        /**
+         *  @deprecated Use copyFiles() instead
+         */
+        @Mutation(() => CopyFilesResponse, { deprecationReason: "Use copyFiles instead" })
         @SkipBuild()
         async copyFilesToScope(
```

Comment:
IMO `copyFilesToScope` is a bad name because you actually copy to a folder (and use the scope of this target folder).

So I deprecated it and replaced it with the more flexible `copyFiles` mutation